gulpfile.js

```diff
@@ -2,17 +2,82 @@
 var gulp = require('gulp');
 var validatePipeline = require('pipeline-validate-js')();
 var validateCssPipeline = require('pipeline-validate-css')();
+var concat = require('gulp-concat');
+var concatcss = require('gulp-concat-css');
+var uglify = require('gulp-uglify');
+var cssnano = require('gulp-cssnano');
+var mainBowerFiles = require('main-bower-files');
+var htmlreplace = require('gulp-html-replace');
 
+//check for css and js errors
 gulp.task('js:lint', function() {
   return gulp
     .src('./src/js/**/*.js')
     .pipe(validatePipeline.validateJS());
 });
-
 gulp.task('css:lint', function() {
   return gulp
-      .src('./src/css/**/*.js')
-      .pipe(validateCssPipeline.validateCSS());
+    .src('./src/css/**/*.css')
+    .pipe(validateCssPipeline.validateCSS());
+});
+
+//concatenate files for venders and app
+gulp.task('js:concatBower', function () {
```

Comment:
Task names are typically lowercase. So in this case, how do you feel about renaming to something like `js:concat:bower`?